docs/1.getting-started/6.data-fetching.md

```diff
@@ -8,9 +8,9 @@ Nuxt comes with two composables and a built-in library to perform data-fetching
 
 In a nutshell:
 
-- [`useFetch`](/docs/api/composables/use-fetch) is the most straightforward way to handle data fetching in a component setup function.
-- [`$fetch`](/docs/api/utils/dollarfetch) is great to make network requests based on user interaction.
-- [`useAsyncData`](/docs/api/composables/use-async-data), combined with `$fetch`, offers more fine-grained control.
+- [`$fetch`](/docs/api/utils/dollarfetch) is the simplest way to make a network request.
+- [`useFetch`](/docs/api/composables/use-fetch) is the SSR-friendly version of `$fetch` that prevent hydration mismatches.
```

Comment:
Might be good to add that it also will not fetch data twice (on server and on client) but just once

Reply:
Thanks for the review, a good point! I've amended that line and it is clearer now.